snakemake/executors/__init__.py

```diff
@@ -178,7 +184,28 @@ def print_job_error(self, job, msg=None, **kwargs):
         job.log_error(msg, **kwargs)
 
     def handle_job_success(self, job):
-        pass
+        if provenance_manager.is_active():
+            prov_mgr = provenance_manager
+            # print(job.params['biotools_id'])
+            tool_name = ""
+            if "biotools_id" in job.params.keys():
+                tool_name = job.params["biotools_id"]
```

Comment:
Mhm, is it possible to extract the biotools_ids from the conda packages instead?